swagger/public_v1/api_spec.json

```diff
@@ -140,7 +140,7 @@
           },
           "findable": {
             "type": "boolean",
-            "description": "Is this course currently visible on the Find Postgraduate Teacher Training service.",
+            "description": "Is this course published on UCAS and have at least one location that is running?",
```

Comment:
I'm not keen on the wording....

Particularly `UCAS`

Reply:
I've dropped the commit for now. I'll leave the wording to you and the Publish / Register crew!